Source/Scene/Cesium3DTileset.js

```diff
@@ -788,19 +816,57 @@ define([
         }
     }
 
-    function getScreenSpaceError(geometricError, tile, frameState) {
+    var scratchPositionNormal = new Cartesian3();
+
+    function updateDynamicScreenSpaceError(tileset, frameState) {
+        var camera = frameState.camera;
+        var positionCartographic = camera.positionCartographic;
+        var height = positionCartographic.height;
+
+        // The range where the density starts to lessen
```

Comment:
How did you come up with these and should they be based on the tileset's bounding box?